src/renderer/prefComponents/imageUploader/index.vue

```diff
@@ -108,11 +126,30 @@ export default {
       })
     },
     setCurrentUploader (value) {
+      const service = services[value]
+      if (!service) {
+        console.error(`Cannot find service "${value}"!`)
+        return
+      }
+
+      const legalNotices = document.getElementsByClassName(`pref-cb-legal-notices ${value}`)
+      if (!legalNotices) {
+        console.error(`Cannot find legal notices of service "${value}"!`)
+        return
+      }
+      legalNotices[0].classList.remove('error')
```

Comment:
how about use `class` attribute in template to instead call dom API directly in vue file.

```html
:class="[{'error':  agreedToLegalNotices}]"
```

but we do not need to show `error` class at the created lifecycle of component, so maybe we need another variable.